frontend/public/locales/de/common.json

```diff
@@ -290,5 +290,6 @@
   "browse": "Durchsuchen",
   "setup-flathub": "Flathub einrichten",
   "setup-flathub-description": "Richte Flathub ein, um Anwendungen auf deinem System zu installieren und zu aktualisieren.",
-  "explore-applications": "Anwendungen entdecken"
+  "explore-applications": "Anwendungen entdecken",
```

Comment:
Please remove all translations, that arenot the `en` ones. Unfortunatly we can't touch them here, as far as I know, or it will need to conflicts - which might cause me a bunch of work :)

Reply:
Removed